src/lib/use-persons-persons.ts

```diff
@@ -0,0 +1,65 @@
+import { groupByToMap } from '@stefanprobst/group-by'
+import { keyByToMap } from '@stefanprobst/key-by'
+import { useMemo } from 'react'
+
+import { usePersonPersonRelations, usePersons } from '@/api/ica.client'
+import { collection } from '~/config/ica.config'
+
+// FIXME:
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+export function usePersonsPersons() {
```

Comment:
what we want is:
(i) request all person-person relations for a set of person ids (all persons in the `webclient` collection), irrespective of whether the person is in the source of target position (i.e. requeset both with `related_personA__id__in=` and `related_personA__id__in=`
(ii) map over resulting relations, discard all relations to persons *not* in the `webclient` collection
(iii) return relations mapped by person id